### ISSUE TITLE
Focus on UI element without activating

### ISSUE THREAD

**Issue by LawranceFung** [zsims/hunt-and-peck#32]
Would it be possible to focus on a UI element without activating it, similar to the 'LinkHints.activateMode action=focus' command in Vimium?  It would be useful if you want to activate the UI element multiple times with the enter key, or trigger the context menu.

**Comment by LaPeste** [comment]
Sorry for the silly question, but is this related to the fact that in general at first `alt+;` the badges don't appear anywhere beside the title?

**Comment by LawranceFung** [comment]
I don't think so.  I think that the hints failing to appear anywhere beside the title initially is because the document tree has yet to be parsed or refresh its visibility.  I'm asking for a feature to focus on actionable elements without clicking on them.
